inksdk/ink/src/test/java/com/microsoft/device/ink/ExampleUnitTest.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+package com.microsoft.device.ink
```

Comment:
If for now there are not going to be unit tests, you can delete this file since adds just noise